lib/trento/activity_log.ex

```diff
@@ -67,6 +77,17 @@ defmodule Trento.ActivityLog do
     end
   end
 
+  # for usage in tests only
+  def user_management_log_types do
+    @user_management_log_types
+  end
+
+  defp scope(ActivityLog = q, true = _user_has_ability?), do: q
```

Comment:
Please rename this function to something more meaningful, like `maybe_with_user_logs bla bla` but not scope